### PR TITLE
Add temporaryCacheLocation to Cache Options (#5725)

### PR DESCRIPTION
**Pull request by tnorling**
This PR ports changes from community PR #5725 into v3

### CODE REVIEW

**Review by peterzenz — APPROVED** [review]

File: lib/msal-browser/docs/configuration.md [hunk 1]
```diff
@@ -88,6 +89,7 @@ const msalInstance = new PublicClientApplication(msalConfig);
 | Option | Description | Format | Default Value |
 | ------ | ----------- | ------ | ------------- |
 | `cacheLocation` | Location of token cache in browser. | String value that must be one of the following: `"sessionStorage"`, `"localStorage"`, `"memoryStorage"` | `sessionStorage` |
+| `temporaryCacheLocation` | Location of temporary token cache in browser. This option is rarely needed to be configured. Please refer to [caching](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/caching.md#cached-artifacts) for more. | String value that must be one of the following: `"sessionStorage"`, `"localStorage"`, `"memoryStorage"` | `sessionStorage` |
```
Comment (hectormmg) [inline lib/msal-browser/docs/configuration.md]: Is it right to call it "token cache" here?

**Review by hectormmg — APPROVED** [review]
Just a comment on the documentation, code looks good.

File: lib/msal-browser/docs/configuration.md (hunk 1 above)
Comment (tnorling) [inline lib/msal-browser/docs/configuration.md]:
```suggestion
| `temporaryCacheLocation` | Location of temporary cache in browser. This option should only be changed for specific edge cases. Please refer to [caching](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/caching.md#cached-artifacts) for more. | String value that must be one of the following: `"sessionStorage"`, `"localStorage"`, `"memoryStorage"` | `sessionStorage` |
```

Comment (tnorling) [reply lib/msal-browser/docs/configuration.md]: Thanks, updated!

**Pull request merged**